src/ExaTioga.h

```diff
@@ -32,14 +32,17 @@ class ExaTioga
     void check_solution_norm();
 
 private:
-    void run_timesteps();
+    void run_timesteps(size_t ofileID);
 
     stk::ParallelMachine m_comm;
 
     tioga_nalu::StkIface m_stk;
     TiogaAMRIface m_amr;
 
     TIOGA::tioga& m_tioga;
+
+    bool m_doWrite;
+    int m_outputFreq;
```

Comment:
```suggestion

    int m_output_freq{1};
    bool m_do_write{true};
```

Reply:
@sayerhs This has be rectified in 
eac195f